docs/src/references/certificate-management.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Certificate Management
+tags: [Reference, Certificate, Security]
+sidebar_position: 1
+description: Certificate Management reference guide
+---
+
+import UserContext from '@site/src/components/UserContext';
+import UserContextForm from '@site/src/components/UserContextForm';
+
+:::tip
+#### User Context {#user-context}
+
+You can customize the documentation and commands shown on this page by providing
+relevant settings which will be reflected in the instructions. It makes it even
+easier to explore and use %%te%%.
+
+<UserContextForm settings="C8Y_PROFILE_NAME,C8Y_PROFILE_URL,C8Y_URL,DEVICE_ID" />
+
+The user context will be persisted in your web browser's local storage.
+:::
+
+## Integration with Cumulocity CA {#cumulocity-certificate-authority}
```

Comment:
A one-line description of the feature is missing.

Reply:
hm, this is getting a bit difficult to describe as the page is definitely 95% c8y focused, so I'm questioning the overall title of the page now.

Reply:
It's currently c8y focused because only that CA is supported for now. Somehow this is what the intro can tell.

I will come with a proposal.

Reply:
See https://github.com/thin-edge/thin-edge.io/pull/3590/commits/718cf509ff4da9aac707a429ac12ddec086c09d2